resources/refund.js

```diff
@@ -22,4 +22,20 @@ function Refund(shopify) {
 
 assign(Refund.prototype, pick(baseChild, ['get', 'buildUrl']));
 
+
+/**
+ * Calculates a refund.
+ * https://help.shopify.com/api/reference/refund#calculate
+ *
+ * @param {Number} parentId Parent record ID
+ * @param {Object} [params] Query parameters
+ * @return {Promise} Promise that resolves with the result
+ * @public
+ */
+Refund.prototype.calculate = function complete(orderId, params) {
+  const url = this.buildUrl(orderId, '/calculate');
+  return this.shopify.request(url, 'POST', undefined, params)
+    .then(body => body[this.key]);
```

Comment:
Remove this line, it's not needed if you add `this.key` as suggested in the previous comment.